src/App/Fossa/Main.hs

```diff
@@ -56,11 +56,14 @@ opts =
   CmdOptions
     <$> switch (long "debug" <> help "Enable debug logging")
     <*> strOption (long "basedir" <> short 'd' <> metavar "DIR" <> help "Set the base directory for scanning (default: current directory)" <> value ".")
-    <*> urlOption (long "endpoint" <> metavar "URL" <> help "The FOSSA API server base URL" <> value (https "app.fossa.com"))
+    <*> urlOption (long "endpoint" <> metavar "URL" <> help "The FOSSA API server base URL" <> value urlOpts)
     <*> optional (strOption (long "project" <> help "this repository's URL or VCS endpoint (default: VCS remote 'origin')"))
     <*> optional (strOption (long "revision" <> help "this repository's current revision hash (default: VCS hash HEAD)"))
     <*> comm
     <**> helper
+    where
+      baseUrl = https "app.fossa.com"
+      urlOpts = UrlOption baseUrl $ mempty
```

Comment:
I'd recommend running hlint over this -- e.g., this `$` is redundant

Reply:
Oops. Yeah, I missed that one. Thanks!

BTW, fun alias:

```
alias hlintdiff="git diff --numstat master... | awk '{print \$3}' | xargs hlint --color=always | less"
```